redfish/session.go

```diff
@@ -6,7 +6,6 @@ package redfish
 
 import (
 	"net/url"
-
```

Comment:
There still needs to be this separation between the std library import group and local imports.